### ISSUE TITLE
Bug while importing GPX file

### ISSUE THREAD

**Issue by Cappa-d** [cgeo/cgeo#4638]
Something must have changed with the last release as I encounter strange errors while importing some of the GPX PQs which was not occuring before.

Actions performed (in that sequence):

- 2 defined PQs (1 with 1k caches - Multi, Tradi, Myst & 1 with the rest - around 3xx EC, WIG, Event etc) range 50 km around my HZ, scheduled weekly
- downloaded via Pocket Query Creator
- one folder created for each PQ
- Import the one with 1k caches = failure
- Import the smaller one = failure
- close c:geo and restart
- Import small PQ --> success
- Import large PQ --> failure
- close c:geo and restart
- Import large PQ --> success

Message shown:
- Import failed - Unexpected error - disk I/O error (code 1802) --> Large PQ
- Import failed - Unexpected error - cannot rollback - no transaction is active (code 1) -- Small PQ
- Result - 324 caches imported -- Small PQ
- Import failed - Unexpected error - cannot rollback - no transaction is active (code 1) -- Large PQ

Device Used:
--- System information ---
Device: GT-I9195 (serranoltexx, samsung)
Android version: 4.4.2
Android build: KOT49H.I9195XXUCNJ4
Cgeo version: 2015.01.20
Play services: present
Use Play services: yes
Low power mode: inactive
Compass capabilities: yes
Rotation sensor: present
Geomagnetic rotation sensor: absent
Orientation sensor: present
--- End of system information ---

**Comment by Lineflyer** [comment]
Can you provide us the GPX files used?

**Comment by Cappa-d** [comment]
send it to the support mail or how should I provide it?

**Comment by samueltardieu** [comment]
Yes, support mail (with reference to this issue #4638) would be great.

**Comment by Cappa-d** [comment]
You got mail ;)

**Comment by Lineflyer** [comment]
Need some more days to have time to test this.
If someone else is keen to do that, feel free.

**Comment by Cappa-d** [comment]
Note: the PQs resulting in these problems are usually created each wednesday. I will test again this week and see if there is a change. Will update here if it continues or stopped.

**Comment by Lineflyer** [comment]
I tested this scenario with 2015.02.04-NB using the files provided via mail.
I could not reproduce any of the mentioned problems on several tries of importing (initially and overwriting on exiting list of caches). All works normal here.
@Cappa-d 

- Any other clues (e.g. did you refresh the caches after import and later on overwrite them with a new GPX-import)?
- if the issue is still present for you, can you test with the newest nightly build available on http://www.cgeo.org/c-geo-nightly.apk ?

**Comment by Cappa-d** [comment]
I tried today with the new PQ from wednesday and the same error occured. Will check with the nightly build now that I am back home.

**Comment by Cappa-d** [comment]
Ok I will close the topic after the tests today. But in fact I am only closing because it seems this problem occured because my c:geo installation on my device was pretty much f*!* up.

I tried the Nightly Build yesterday and there was no change
I installed todays nightly build and afterwards c:geo was simply unusable. It crashed every other try. So in the end I decide same the DB and kill it.
New install and check than --> all fine again.

Whatever was the cause I do not know but it works again without crashing but I am back to the currently released version.
